napari/_tests/test_viewer.py

```diff
@@ -95,26 +103,28 @@ def test_add_layer(make_napari_viewer, layer_class, data, ndim):
         func(layer)
 
 
-EXPECTED_NUMBER_OF_LAYER_METHODS = {
-    'Image': 5,
-    'Vectors': 0,
-    'Surface': 0,
-    'Tracks': 0,
-    'Points': 9,
-    'Labels': 14,
-    'Shapes': 17,
-}
+layer_types = (
+    'Image',
+    'Vectors',
+    'Surface',
+    'Tracks',
+    'Points',
+    'Labels',
+    'Shapes',
+)
 
 
-@pytest.mark.parametrize(
-    'cls, expectation', EXPECTED_NUMBER_OF_LAYER_METHODS.items()
-)
-def test_expected_number_of_layer_methods(cls, expectation):
+@pytest.mark.parametrize('layer_class, data, ndim', layer_test_data)
+def test_all_layer_actions_are_accessible_via_shortcut(
+    layer_class, data, ndim
+):
     """
-    Make sure we do find all the methods attached to a layer via keybindings
+    Make sure we do find all the actions attached to a layer via keybindings
     """
-    layer_methods = _get_all_keybinding_methods(getattr(layers, cls))
-    assert len(layer_methods) == expectation
+    _ = layer_class(
```

Comment:
Why assignment to `_`?

Reply:
we just need to initialize it, but it's otherwise useless.